manual/manual.html

```diff
@@ -7880,11 +7889,33 @@ <h3 id="_words_identifiers_and_names"><a class="anchor" href="#_words_identifier
 </div>
 </div>
 <div class="paragraph">
-<p>A quoted identifier starts and ends with single quotes and may contain any character except quotes (including spaces).
-By quoting any sequence of characters can become an identifier.
-A quoted identifier may also be used to make an identifier out of a reserved word such as <code>Look</code>.
-This is useful in the definition of the verb <code>look</code>.
-It would look like:</p>
+<p>A quoted identifier starts and ends with single quotes and may contain any character (including spaces) except quotes.
+Any sequence of characters enclosed within single quotes can become an identifier, except inside strings (where single quotes are treated as ordinary printable characters).</p>
+</div>
+<div class="admonitionblock note">
+<table>
+<tr>
+<td class="icon">
+<i class="fa icon-note" title="Note"></i>
+</td>
+<td class="content">
+<div class="paragraph">
+<p>A single-word identifier can be written within or without single quotes, indifferently (unless it&#8217;s a keyword).
```

Comment:
"indifferently" -> "the quotes in themselves having no significance."?

Reply:
Yes, if it's a one-word identifier (no spaces) which is not a reserved keyword (i.e. not requiring stropping) than one is free is to use it with or without quotes, for the compiler will see them as references to the same ID.

Reply:
Practical example:

```alan
The kitchen IsA location
  Has magicspell "Avra kaDavra".
End the.

Start at kitchen.
Say 'magicspell' of kitchen. --> 'magicspell' == magicspell
```

Reply:
Yeah. I know. I actually meant is the choice of "indifferently" really what we mean. Doesn't that refer to feelings, rather than difference. I'm not a native English speaker but to me "indifferently" sounds like "I don't care". 

What we mean is that it doesn't make any difference if quotes are used or not, if they are not required for some other reason. My comment was a suggestion for different wording.

Reply:
> Doesn't that refer to feelings, rather than actual non-difference. 

It could mean both, but in this context is rather clear that it means that there are no consequences for choosing one over the other (or mixing them freely in the same code).
Since it refers to the act of writing, we know that no personal feelings are involved. (😂, 😡, etc.)